lib/mix/phoenix/schema.ex

```diff
@@ -158,6 +170,8 @@ defmodule Mix.Phoenix.Schema do
     Enum.map(attrs, fn attr ->
       attr
       |> drop_unique()
+      |> drop_redact()
+      |> drop_unique()
```

Comment:
This makes me think we should probably refactor the code so we do a pre-processing on all attributes collecting redact/unique. WDYT?

Reply:
@josevalim I rewrote the code to pre-process as you suggested. It both collects the lists of uniques/redacts and removes them from the cli_attrs so subsequent calls don't have to all manually ignore them.